star.wdl

```diff
@@ -1,47 +1,27 @@
 version 1.0
 
-struct InputGroup {
-  File fastqR1
-  File fastqR2
-  String readGroup
-}
-
 workflow star {
-  input {
-    Array[InputGroup] inputGroups
-    String outputFileNamePrefix
-  }
-
-  scatter (ig in inputGroups) {
-    File read1s       = ig.fastqR1
-    File read2s       = ig.fastqR2
-    String readGroups = ig.readGroup
-  }
-
-  parameter_meta {
-    inputGroups: "Array of fastq files to align with STAR and the merged filename"
-    outputFileNamePrefix: "Prefix for filename"
-  }
+input {
+ Array[Pair[Pair[File, File], String]]+ inputFqsRgs
+ String outputFileNamePrefix
+}
 
-  call runStar {
-    input:
-    read1s = read1s,
-    read2s = read2s,
-    readGroups = readGroups,
-    outputFileNamePrefix = outputFileNamePrefix
-  }
+scatter (FqRg in inputFqsRgs) {
+    File read1s       = FqRg.left.left
+    File read2s       = FqRg.left.right
+    String readGroups = FqRg.right
+}
 
-  call indexBam {
-   input:
-   inputBam = runStar.outputBam }
+call runStar { input: read1s = read1s, read2s = read2s, readGroups = readGroups, outputFileNamePrefix = outputFileNamePrefix }
+call indexBam { input: inputBam = runStar.outputBam }
 
-  meta {
-   author: "Peter Ruzanov, Alexander Fortuna"
-   email: "peter.ruzanov@oicr.on.ca, alexander.fortuna@oicr.on.ca"
-   description: "STAR 2.1"
-   dependencies: [
+meta {
+ author: "Peter Ruzanov"
+ email: "peter.ruzanov@oicr.on.ca"
+ description: "STAR 2.0"
+ dependencies: [
       {
-        name: "star/2.7.6a",
+        name: "star/2.7.3a",
```

Comment:
I am confused. The CHANGELOG says: update to 2.7.6a, but the version in the commit is 2.7.3a? README also says 2.7.6a, something needs to be adjusted

Reply:
This is the item that I added in the changelog
## 2.0.4 - 2022-06-22
- migrate star 2.0.2 to Vidarr